src/code/game.c

```diff
@@ -1,6 +1,12 @@
 #include "global.h"
 #include "terminal.h"
 
+#ifdef OOT_DEBUG
+    #define VI_MODE_EDIT_STATE_COND (R_VI_MODE_EDIT_STATE == VI_MODE_EDIT_STATE_INACTIVE)
+#else
+    #define VI_MODE_EDIT_STATE_COND true
+#endif
```

Comment:
I don't think this should be called `_COND`, that's a bit redundant. Could we give this a more descriptive name like `VI_MODE_ENABLED`? (Maybe wait until another reviewer has a stronger opinion on the name)

Reply:
I named it like this because I had no idea how to call it lol
I don't have enough knowledge about the VI mode to say if your suggestion is a good name though, but it sounds better than my original idea

Reply:
`VI_MODE_EDITOR_INACTIVE`?